front: 0
left: 1
back: 2
right: 3

view_2d: True
plot_graphs: False
save_data: True

obstacle: 0
drone: 1
floor: 2

node: 'node'
name: 'name'
color: 'color'
past_xy: "past_xy"
past_time: "past_time"
agent: "agent"
option: "option"
forces: "forces"
des_v: "des_vel"
obj_detected: "obj_detected"

# Communication keys
scope_pose: 'POSE'
scope_dists: 'DISTS'
scope_yaw_rate: "YAWRATE"
scope_velocity: "VEL"
scope_controls: "CONTROLS"